geometric/optimize.py

```diff
@@ -1502,10 +1503,10 @@ def run_optimizer(**kwargs):
             else:
                 xyzout = prefix+".xyz"
                 xyzout2="opt.xyz"
-            coords = Optimize(coords, M, IC, engine, dirname, params, xyzout,xyzout2)
+            progress = Optimize(coords, M, IC, engine, dirname, params, xyzout,xyzout2)
             print
     print_msg()
-    return M
+    return progress
```

Comment:
Here the return value of run_optimize() is also changed to progress, and I feel that previously returning the `M` with initial coordinates might be confusing.